RASP_support/Evaluator.py

```diff
@@ -224,7 +224,7 @@ def _evaluateDictComp(self, ast):
 		if not (isinstance(d, list) or isinstance(d, dict)):
 			raise RASPTypeError(
 				"dict comprehension should have got a list or dict to loop "
-				+ "over, but got:", l)
+				+ "over, but got:", d)
```

Comment:
good catch but i actually was hoping to get this in a separate pr cause it fixes an actual bug